src/elife_profile/modules/custom/elife_resources/elife_resources.class.inc

```diff
@@ -76,7 +76,7 @@ class elifeResources {
       return $response;
     }
     else {
-      $response = drupal_http_request(self::RESOURCES_API . $this->query);
+      $response = drupal_http_request(self::RESOURCES_API . $this->query, ['timeout' => 3]);
```

Comment:
it would be useful to have this `3` as a constant defined somewhere to change it in a single place when needed